{"name": "Проверка сообщения: Неверные учетные данные пользователя при неуспешной авторизации с неверным логином", "status": "passed", "parameters": [{"name": "login", "value": "'admin1'"}, {"name": "password", "value": "'zj#KqU1'"}], "start": 1654271078174, "stop": 1654271080122, "uuid": "256c888a-85b3-48b7-bc13-17af19631ac2", "historyId": "a292f92d2ce761f5f0b012d85720427b", "testCaseId": "3cb536bbda2aa2103ba5594cfd411340", "fullName": "test_autorization#test_wrong_login", "labels": [{"name": "severity", "value": "minor"}, {"name": "suite", "value": "test_autorization"}, {"name": "host", "value": "DESKTOP-13RCMAF"}, {"name": "thread", "value": "9984-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_autorization"}]}